track_local_static.go

```diff
@@ -279,9 +293,25 @@ func (s *TrackLocalStaticSample) WriteSample(sample media.Sample, onRtpPacket fu
 	if sample.PrevDroppedPackets > 0 {
 		p.(rtp.Packetizer).SkipSamples(samples * uint32(sample.PrevDroppedPackets))
 	}
-	packets := p.(rtp.Packetizer).Packetize(sample.Data, samples)
 
-	err := addExtensions(sample, packets)
+	payloadDataIdx := -1
+	var packets []*rtp.Packet
+
+	if s.hyperscaleEncryption {
+		packets, payloadDataIdx = p.(rtp.Packetizer).PacketizeAndDetectData(sample.Data, uint32(samples))
```

Comment:
Notice there are build failures..
You should probably about rtp version in go.mod & go.sum
(This repo contains build & test action which we should ensure keep passing)
Will be a good idea to add tests as well